### ISSUE TITLE
Error for message missing from email address

### ISSUE THREAD

**Issue by atindale** [JensAstrup/pyOutlook#7]
Hi, I'm getting a KeyError when reading emails using get_inbox().

```
Traceback (most recent call last):
  File "read_mail2.py", line 30, in <module>
    inbox = account_one.get_inbox()
  File "C:\Users\Alan\Projects\read_email\venv\lib\site-packages\pyOutlook\core\main.py", line 90, in get_inbox
    return retrieve.get_inbox(self)
  File "C:\Users\Alan\Projects\read_email\venv\lib\site-packages\pyOutlook\internal\retrieve.py", line 66, in get_inbox
    return clean_return_multiple(r.json())
  File "C:\Users\Alan\Projects\read_email\venv\lib\site-packages\pyOutlook\internal\retrieve.py", line 22, in clean_return_multiple
    sender_email = key['Sender']['EmailAddress']['Address']
KeyError: 'Address'
```

Investigated showed that I had a email from Microsoft Outlook Calendar with the subject _Daily schedule on Sunday, 1 January 2017_. This email did not include a from email address; just a name. The JSON is below.

```
    'From': {
        'EmailAddress': {
            'Name': 'Microsoft Outlook Calendar'
        }
    },
```

To fix the logic below could be added to `clean_return_multiple()`.

```
            if 'Address' in key['Sender']['EmailAddress']:
                sender_email = key['Sender']['EmailAddress']['Address']
            else:
                sender_email = "None"
```

**Comment by JensAstrup** [comment]
[Fixed in commit for v3.3.1b](https://gitlab.com/jensastrup/pyOutlook/commit/02a28c6c8e6fad64470452e04dac886810654eea)

**Comment by JensAstrup** [comment]
Fixed in #8
